src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -1552,4 +1545,127 @@ public int cullInvalidConsumerContentAccess(Owner owner, String... existingModes
         return entityManager.createQuery(query).getResultList();
     }
 
+    @Transactional
+    private int postgreUnlinkIdCertificates(Collection<String> certIds) {
```

Comment:
In all of these backend-specific queries, we're doing an UPDATE-JOIN on the primary key (cert ID), and then examining the ID of the cert to select which rows to update.

Unless I'm missing something, the join is unnecessary -- especially if we're stepping into raw SQL.

Reply:
Done.